components/NoAgentNotification.tsx

```diff
@@ -67,10 +67,14 @@ export function NoAgentNotification(props: NoAgentNotificationProps) {
             </svg>
           </div>
           <p className="text-pretty w-max">
-            It&apos;s quiet... too quiet. Is your agent lost? Make sure your
-            agent is set up and running correctly.
+            It&apos;s quiet... too quiet. Is your agent lost? Ensure your agent
+            is properly configured and running on your machine.
           </p>
-          <a href="#" className="underline whitespace-nowrap">
+          <a
+            href="https://github.com/livekit-examples/voice-assistant-frontend/blob/main/README.md"
```

Comment:
```suggestion
            href="https://github.com/livekit-examples/voice-assistant-frontend?tab=readme-ov-file"
```

Reply:
Nicer to land you on the repo homepage, and this isn't coupled to any branch name